src/_P036_FrameOLED.ino

```diff
@@ -780,16 +787,15 @@ void display_time() {
 void display_title(String& title) {
   display->setFont(ArialMT_Plain_10);
   display->setColor(BLACK);
-//  display->fillRect(0, 0, 128, 13); // Underscores use a extra lines, clear also.
-  display->fillRect(0, TopLineOffset, 128, 12);   // don't clean line under title.
+  display->fillRect(0, TopLineOffset, 128, 12);   // don't clear line under title.
```

Comment:
Is this 128 a given for all these controllers, or should this also be P36_MaxDisplayWidth ?